pkg/controller/infinispan/infinispan_controller.go

```diff
@@ -1416,3 +1442,15 @@ func getEnvWithDefault(name, defVal string) string {
 	}
 	return defVal
 }
+
+// getServingCertsMode returns a label that identify the kind of serving
+// certs service is available. Returns 'openshift.io' for service-ca on openshift
+func getServingCertsMode(remoteKubernetes *ispnutil.Kubernetes) string {
+	if remoteKubernetes.HasServiceCAsCRDResource() {
+		return "openshift.io"
```

Comment:
Why return this string? Seems a little arbitrary and not sure what other situation you might have in mind where this method would return something other than `""` or `"openshift.io".

Couldn't you just skip this method and ask `remoteKubernetes` if openshift service CA exists, and use that in the caller in this function?

Reply:
This is to easily add other certs service in the future. Service discovery can be isolated in this method instead of mix the code with the reconciliaiton stuff.
String labels are needed instead of bool because different serviced may require different reconciliation policies in the caller.

Reply:
@galderz if the comments are ok, can we go on and merge?